src/endpoint/sts/ops/sts_post_assume_role.js

```diff
@@ -0,0 +1,51 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const dbg = require('../../../util/debug_module')(__filename);
+const { StsError } = require("../sts_errors");
+
+/**
+ * https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html
+ */
+async function assume_role(req) {
+    dbg.log0('sts_post_assume_role parmas: ', req.params);
+    let assumed_role;
+    try {
+        assumed_role = await req.sts_sdk.get_assumed_role(req);
+    } catch (err) {
+        if (err.code === 'ACCESS_DENIED') {
+            throw new StsError(StsError.AccessDeniedException);
+        }
+        throw new StsError(StsError.InternalFailure);
+    }
+    // Temporary credentials are NOT stored in noobaa
+    // TODO: need to generate session token and store in it 
+    // the temporary credentials and expiry
```

Comment:
Shouldn't we handle this TODO for anything to work?

Reply:
currently i'm just returning access keys, will generate and parse the session token in the following PR.

Reply:
but if these are temp credentials that are not stored anywhere - how can you validate requests with those creds?

Reply:
maybe I'm missing something here..
what I'm saying is that after this PR will be merged the temporary credentials that a user will get as a response of assumeRole can not be used and will be denied. After merging the changes of the next PR the credentials can be accapted.

Reply:
Ah OK, I see now that this PR will not yet be usable until the next one. Thanks for clarifying